scripts/get-okteto-beta.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+{ # Prevent execution if this script was only partially downloaded
+
+        set -e
+
+        install_dir='/usr/local/bin'
+        install_path='/usr/local/bin/okteto'
+        OS=$(uname | tr '[:upper:]' '[:lower:]')
+        ARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
+        gh_api_url='https://api.github.com/repos/okteto/okteto/releases'
+        download_url='https://github.com/okteto/okteto/releases/download/'
+        cmd_exists() {
+                command -v "$@" >/dev/null 2>&1
+        }
+
+        if ! cmd_exists curl; then
+                printf '\033[31m> The command "curl" is not found and is required by this installation script.\n\033[0m'
+                exit 1
+        fi
+
+        if ! cmd_exists jq; then
+                printf '\033[31m> The command "jq" is not found and is required by this installation script.\n\033[0m'
+                exit 1
+        fi
+
+        latestRelease=${download_url}$(curl -s ${gh_api_url} | jq --raw-output '.[0].tag_name')
```

Comment:
I guess, the default script must be updated to install the latest official release

Reply:
No, the script calls this url: [https://github.com/okteto/okteto/releases/latest/download](https://github.com/okteto/okteto/releases/latest/download) which is always pointing to the latest release (not pre-release), and the new one uses GitHub api + jq to get the latest regardless of whether it is pre-release or not